### PR TITLE
Fixes broken doc build 

### PR DESCRIPTION
**Pull request by awholegunch**
and fixes check style error in documentation strings.

Running Doc Build: http://builds.cask.co/browse/CDAP-DDBD540-1/log

Results staged at: [Examples](http://docs-staging.cask.co/cdap/3.1.0-SNAPSHOT-feature-r3.1-fix_broken_doc_build/en/examples-manual/examples/index.html)

Passes checkstyle (locally).

### CODE REVIEW

File: cdap-docs/application-templates/source/etl/templates/sources/real-time/index.rst [hunk 1]
```diff
@@ -11,6 +11,5 @@ Sources: Real-time
 
     JMS <jms>
     Kafka <kafka>
-    SQS <sqs>
```
Comment (sreevatsanraman) [inline cdap-docs/application-templates/source/etl/templates/sources/real-time/index.rst]: Why is this removed?

Comment (sreevatsanraman) [inline cdap-docs/application-templates/source/etl/templates/sources/real-time/index.rst]: it should be Amazon SQS

Comment (awholegunch) [inline cdap-docs/application-templates/source/etl/templates/sources/real-time/index.rst]: We'll need to talk to Albert. He removed it in this PR (https://github.com/caskdata/cdap/pull/3365), though I see that the java file (...p-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/SqsSource.java) was still in the PR.

https://github.com/caskdata/cdap/pull/3365/files#diff-a584fe18788dda704f586682d79f70fd

Probably needs to be restored and re-vised, unless there is some issue with exposing it.

Comment (albertshau) [inline cdap-docs/application-templates/source/etl/templates/sources/real-time/index.rst]: should be renamed to amazonsqs, since ui will just be using the name of the plugin.  sorry forgot to update it here

Comment (albertshau) [inline cdap-docs/application-templates/source/etl/templates/sources/real-time/index.rst]: aaaand i forgot to git add it..

Comment (awholegunch) [inline cdap-docs/application-templates/source/etl/templates/sources/real-time/index.rst]: Albert got back to me about the deleted file: that was an error; it should have been a rename..we're working on fixing it.

File: cdap-docs/application-templates/source/etl/templates/sources/real-time/amazonsqs.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+=======================
+Sources: Real-time: SQS
```
Comment (awholegunch) [inline cdap-docs/application-templates/source/etl/templates/sources/real-time/amazonsqs.rst]: Should this be:
```
Sources: Real-time: Amazon SQS

Comment (albertshau) [inline cdap-docs/application-templates/source/etl/templates/sources/real-time/amazonsqs.rst]: yup

**Comment by albertshau** [comment]
lgtm
